pkg/kyc/component.go

```diff
@@ -118,33 +123,78 @@ func (c *component) GetUserByUsernameInSocialRealm(ctx context.Context, username
 		return apikyc.UserRepresentation{}, err
 	}
 
+	var realmConf configuration.RealmAdminConfiguration
+	if realmConf, err = c.configDBModule.GetAdminConfiguration(ctx, c.socialRealmName); err != nil {
+		return apikyc.UserRepresentation{}, err
+	}
+	var consentRequired = realmConf.ConsentRequired == nil || *realmConf.ConsentRequired
```

Comment:
It might be easier for evereyone (i.e. front) if this methods always return minimum data ?
I imagine the angular app will not optimize call according to the consent config ?